resources/content/db/service_discovery.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@cjellick yes, this name is temporary so I don't run into merge conflicts while doing rebase